module/VuFind/src/VuFind/Db/Service/LoginTokenService.php

```diff
@@ -173,6 +233,19 @@ public function getBySeries(string $series): array
      */
     public function deleteExpired(DateTime $dateLimit, ?int $limit = null): int
     {
-        return $this->getDbTable('LoginToken')->deleteExpired($dateLimit->format('Y-m-d H:i:s'), $limit);
+        // Date limit ignored since login token already contains an expiration time.
+        $subQueryBuilder = $this->entityManager->createQueryBuilder();
+        $subQueryBuilder->select('lt.id')
+            ->from($this->getEntityClass(LoginTokenEntityInterface::class), 'lt')
+            ->where('lt.expires < :dateLimit')
+            ->setParameter('dateLimit', time());
```

Comment:
I think this should be:

```suggestion
            ->setParameter('dateLimit', $dateLimit->getTimestamp());
```

(Sorry for leading you astray by mentioning `time()`; I forgot that there was a timestamp being passed in already!

Reply:
(It looks like you resolved this without applying the change -- I'm guessing that was unintentional, so I unresolved the comment and committed the suggestion).